src/porepy/utils/sort_points.py

```diff
@@ -184,16 +184,13 @@ def update_queue(pair_0, pair_1):
             )
         )
         hit_old = np.logical_and.reduce(
-            (
-                is_ordered,
-                np.any(t == q[0], axis=0),
-                np.any(t == q[1], axis=0),
-            )
+            (is_ordered, np.any(t == q[0], axis=0), np.any(t == q[1], axis=0))
```

Comment:
It seems we have slightly different versions of black - both the old and the new versions are obviously automatically formatted, but in different ways.

Anyhow, I can see there is some cleaning to do in this file; I will take care of it.